### ISSUE TITLE
Fabric option?

### ISSUE THREAD

**Issue by nazzyz** [wendall911/ReadyPlayerFun#11]
This is not really an "issue", more of a request?
Thank you for this great mod, this really is the only active updated mod out there and is such an invaluable tool for server ops.  I use this on my servers and have recently spun up a Fabric 1.19 server and would like to use this with it.
I don't know if you have experience working with Fabric, but would you be able to add support for this modloader?
Thank you so much if that would be possible.
-n

**Comment by wendall911** [comment]
I think this is a great idea. This was actually the mod I was going to target for supporting both Forge/Fabric. So tentatively, I'll say yes, but not sure the timeline, but should be relatively soon.

**Comment by nazzyz** [comment]
That is fantastic! My apologies, I hate to be "that guy" who asks when will you make such and such version... etc.  Thanks for the positive reply.

**Comment by wendall911** [comment]
@nazzyz Please take a look in the latest files section: https://www.curseforge.com/minecraft/mc-mods/ready-player-fun/files

Everything tests well for me. Mod can exist on client-side and not cause any issues. It works very well in my testing. I was also able to find and fix a couple bugs with settings not being handled correctly.

**Comment by nazzyz** [comment]
This is amazing - hats off to you for fulfilling this request!  I hate to
be the beggar but the version here is only usable for 1.19.* versions, but
not compatible for base 1.19.  While your mod may probably work with any
version listed above, the dependency is requiring a file that is only made
for these later versions.
"Mod 'Ready Player Fun' (readyplayerfun) 2.4.1.11 requires version 6.1.39
or later of mod 'Architectury' (architectury)
Would you be able to supply a version of RPF that would let us use
architectury-5.12.42 and earlier for base 1.19?
Thank you again for your work!

**Comment by wendall911** [comment]
Oh, that's a good point. I didn't realize anyone would be using 1.19 version. I think it should be fine to relax the requirement to load for all versions. I'll work on this and push something out today for you to test. Thanks for the feedback!

**Comment by wendall911** [comment]
Ok, I pushed out another version that works with 1.19 Fabric and 5.12.42 Architectury:
https://www.curseforge.com/minecraft/mc-mods/ready-player-fun/files/3931220

I tested locally with a 1.19 server and client. Works well!

Thanks again for the feedback on this!

**Comment by nazzyz** [comment]
Thank you sir!  Truly the salt of the earth you are.
I tried the modified version and worked like a charm, fortunately (or
unfortunately) the developer of a major mod I have included in a modpack
theme I have been working on released an update this evening as well as
bumping the version requirements to 1.19.2, which is going to be a headache
for me to work on now... BUT! I can now use your original release for
1.19.2 :D
Again, thank you for your work and time.  Great mod and a great idea you
created here.
-n
